force-app/main/default/classes/SOQL.cls

```diff
@@ -1146,6 +1167,11 @@ public virtual inherited sharing class SOQL implements Queryable {
             return set('IN', joinQuery);
         }
 
+        public Filter notLike() {
```

Comment:
And let's make `notLike` as `private`. There is no need to expose it outside ;>

Reply:
Done